### DISCUSSION TITLE
Is it possible to use virtual disk to save the images?

### DISCUSSION THREAD

**Discussion by EsaJaatinen** [jomjol/AI-on-the-edge-device#2712] [General]
Hi

I had this working well with home assistant, however after a couple of power outs the microsd was damaged and the system doesn't boot anymore.

Is it possible to make the system such, that it only boots from the flash and uses RAM for the images to be read?

A small UPS would solve this, however it's a bit stupid solution imho.

Thanks in advance and kudos for a awesome project!

'ej

**Comment by caco3** [comment]
To reduce the SD card writing, change the log level to warning and disable any images saving.
Reading the SD card will not be an issue.

We cant load everything to RAM because the models are too large for the available RAM.

Best is to use a good quality SD card. I have 2 devices mostly running on INFO but sometimes also on DEBUG log level and never had an issue with a card going bad.

**Comment by dtrumpfheller** [comment]
I am having the same issue with power outages and SD card issues afterwards. My plan is to hook a small power bank in between cam and outlet as some kind of cheap UPS.
